docroot/profiles/custom/cgov_site/modules/custom/cgov_core/src/Plugin/Block/CgovDisqus.php

```diff
@@ -1,7 +1,8 @@
 <?php
 
-namespace Drupal\cgov_blog\Plugin\Block;
```

Comment:
This looks good, but I would like the pieces to be separate commits. So make a separate pr for the core function, then one for the others

Reply:
If the pr for the core is just one commit, then you can easily merge it into your work branches